lua/nvim-lsp-installer/log.lua

```diff
@@ -64,7 +62,7 @@ do
 
     local log_at_level = function(level, level_config, message_maker, ...)
         -- Return early if we're below the current_log_level
-        if level < current_log_level then
+        if level < vim.g.lsp_installer_log_level then
```

Comment:
I did some quick profiling with accessing `vim.g` properties, which I believe is a metatable proxy, and it has negligible perf penalties